assets/src/components/hooks/usePlatform.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react'
+
+const getPlatform = () => {
+  let modKeyString = 'Ctrl'
+  let altKeyString = 'Alt'
+  let shiftKeyString = 'Shift'
+  let isMac = false
+  let keyCombinerString = '+'
+
+  if (
+    window?.navigator?.platform.indexOf('Mac') === 0 ||
+    window?.navigator?.platform === 'iPhone'
+  ) {
```

Comment:
minor style thing, but I'd just return straight from this instead of doing the control flow w/ variable assignments.